duckling/language.py

```diff
@@ -58,3 +58,13 @@ def convert_to_duckling_language_id(cls, lang):
         else:
             raise ValueError("Unsupported language '{}'. Supported languages: {}".format(
                 lang, ", ".join(cls.SUPPORTED_LANGUAGES)))
+
+    @classmethod
+    def convert_to_iso(cls, lang):
```

Comment:
Why aren't you reusing `convert_to_duckling_language_id`? Or replace everything with `convert_to_iso`? `convert_to_iso` and `convert_to_duckling_language_id` are basically the same function.